Resources/Prototypes/_Goobstation/Accents/word_replacements.yml

```
@@ -1,4 +1,87 @@
 # Accents that work off of word replacements.
+- type: accent #(Goobstation) New Yowie Accent, to whoever ends up reading this, remember to spell each word exactly
+  id: bogan
+  wordReplacements:
+    accent-bogan-words-1: accent-bogan-words-replace-1
+    accent-bogan-words-2: accent-bogan-words-replace-2
+    accent-bogan-words-3: accent-bogan-words-replace-3
+    accent-bogan-words-4: accent-bogan-words-replace-4
+    accent-bogan-words-5: accent-bogan-words-replace-5    
```

yamllint

[error] trailing spaces

(trailing-spaces)